packages/web-components/src/radio/README.md

```diff
@@ -0,0 +1,133 @@
+| `disabled`      | public  | `boolean`           |           | Sets disabled state for radio                                                                                                               |
+| `labelPosition` | public  | `"after"` `"below"` | `"below"` | The position of the label relative to the radio indicator.                                                                                  |
+| `checked`       | public  | `boolean`           | `false`   | When true, radio button will be checked                                                                                                     |
+
+<br />
+
+### **Methods**
+
+<br />
+
+### **Events**
+
+| Name     | Type | Inherited From |
+| -------- | ---- | -------------- |
+| `change` |      |                |
+
+<br />
+
+### **Attributes**
+
+| Name             | Field         |
+| ---------------- | ------------- |
+| `name`           | name          |
+| `disabled`       | disabled      |
+| `label-position` | labelPosition |
+| `checked`        | checked       |
+
+<br />
+
+### **Slots**
+
+| Name                | Description                            |
+| ------------------- | -------------------------------------- |
+| `checked-indicator` | The checked indicator                  |
+|                     | Default slotted content for label text |
+
+<br />
+<hr />
+<br />
+
+### **Suggested Template**
+
+<br />
+
+`radioTemplate` from FastFoundation
+
+<br />
+<hr />
+<br />
+
+## **Accessibility**
+
+[W3 Radio Spec](https://www.w3.org/WAI/ARIA/apg/patterns/radio/)
+
+<br />
+
+### **WAI-ARIA Roles, States, and Properties**
+
+| Attributes      | value   | Description                         |
+| --------------- | ------- | ----------------------------------- |
+| `role`          | `radio` |
+| `aria-checked`  |         | the checked state of the component  |
+| `aria-required` |         | the required state of the component |
+| `aria-disabled` |         | the disabled state of the component |
+| `tabindex`      | `0`     |                                     |
+
+<br />
+<hr />
+<br />
+
+## **Preparation**
+
+<br />
+
+### **Fluent Web Component v3 v.s Fluent React 9**
+
+<br />
+
+**Component and Slot Mapping**
+
+| Fluent UI React 9 | Fluent Web Components 3 |
+| ----------------- | ----------------------- |
+| `<Radio>`         | `<fluent-radio>`        |
+
+<br />
+
+**Property Mapping**
+| Fluent UI React 9 | Fluent Web Components 3 | Description of difference |
+|-------------------|------------------------ |---------------------------|
+| `prop label` | default slotted content | React implementation requires user to pass a string through the `label` prop on the Radio component <br /><br /> The web component implementation requires users to pass the label text through the default slotted content
```

Comment:
nit, technically it's not required to pass the text for the label via the default slot; you could pass an aria-label or use aria-labelledby to associate it with an element outside it's slot...but the slot is the primary place where the label would go.